packages/pie/src/Pie.tsx

```diff
@@ -161,7 +160,8 @@ const Pie = ({
     if (enableSliceLabels && layers.includes('sliceLabels')) {
         layerById.sliceLabels = (
             <g key="sliceLabels" transform={`translate(${centerX},${centerY})`}>
-                <SliceLabels
+                <SliceLabels<R>
+                    // @ts-ignore
                     dataWithArc={dataWithArc}
```

Comment:
Why `// @ts-ignore` this line?

Reply:
It should not :/

Reply:
I removed it locally and it compiled fine. Maybe an artifact?

Reply:
Removed.